fission/httptrigger.go

```diff
@@ -57,6 +58,17 @@ func getMethod(method string) string {
 	return ""
 }
 
+func checkFunctionExistence(fissionClient *client.Client, fnName string) {
+	meta := &metav1.ObjectMeta{
+		Name:      fnName,
+		Namespace: metav1.NamespaceDefault,
+	}
+	_, err := fissionClient.FunctionGet(meta)
+	if err != nil {
+		fmt.Printf("function '%v' not exists, use 'fission function create --name %v ...' to create function later\n", fnName, fnName)
```

Comment:
not exists -> does not exist

to create function later -> to create the function